frontend/src/app/features/dashboard/task-form/task-form.component.ts

```diff
@@ -1,42 +1,55 @@
-import * as dashboardActions from '@store/actions/dashboard.actions';
-
 import { AbstractControl, FormControl, FormGroup, ValidatorFn, Validators } from '@angular/forms';
-import { Component, OnInit } from '@angular/core';
-import { differenceInDays, endOfDay, startOfDay } from 'date-fns';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { addDays, differenceInDays, endOfDay, startOfDay } from 'date-fns';
 
 import { BaseComponent } from '@shared/components/base/base.component';
+import { DatePipe } from '@angular/common';
 import { NewTask } from '#types/dashboard/tasks/new-task';
-import { Store } from '@ngrx/store';
 import { TaskRepeatingPeriod } from '#types/dashboard/tasks/enums/task-repeating-period';
 import { getRepeatingPeriods } from '@utils/creation-functions/repeating-period.helper';
 import { hasLengthErrors } from '@utils/validators/common-validators';
-import { selectChosenDashboardId } from '@store/selectors/dashboard.selectors';
 import { setTime } from '@utils/dates/dates.helper';
-import { take } from 'rxjs';
 
 @Component({
     selector: 'so-task-form',
     templateUrl: './task-form.component.html',
     styleUrls: ['./task-form.component.sass']
 })
-export class TaskFormComponent extends BaseComponent implements OnInit {
+export class TaskFormComponent extends BaseComponent {
     public readonly TaskRepeatingPeriod = TaskRepeatingPeriod;
+    @Input() public set initialValue(value: NewTask | null) {
```

Comment:
keep some clear ordering, e.g. inputs, outputs, public readonly, public, private readony, private.
decide what is useful more for you. Mix of these all is does not help to observe existing fields

Reply:
Fixed